frontend/resourceadm/components/ResourceReferenceFields/ResourceReferenceFieldset.tsx

```diff
@@ -133,14 +133,17 @@ export const ResourceReferenceFieldset = ({
       >
         <NativeSelect
           size='small'
-          onChange={(event) => setReferenceSource(event.target.value as ResourceReferenceSource)}
+          onChange={(event) => {
```

Comment:
Maybe we can refactor the code in this onChange function into a function outside of the JSX, named ```handleChange``` or something? 😄